docs/tenant.puml

```diff
@@ -0,0 +1,185 @@
+@startuml tenant
+skinparam DefaultTextAlignment center
+skinparam ArrowColor blue
+skinparam ComponentStyle uml1
+skinparam Ranksep 200
+skinparam Linetype ortho
+
+actor "cf user" as user
+
+cloud S3 [
+    ==S3 bucket
+    redundant storage for logs
+]
+cloud ALB [
+    ==app ALB
+]
+cloud UAAALB [
+    ==UAA ALB
+]
+
+rectangle cloudfoundry {
+    node gorouter [
+        ==gorouter
+        dynamic reverse-proxy load balancer
+    ]
+    node nats [
+        ==nats
+        message bus
+    ]
+    node capi [
+        ==CAPI
+        the cf api
+    ]
+    node doppler [
+        ==cloudfoundry doppler
+        this is the source of all the logs
+    ]
+    node uaa [
+        ==UAA
+        auth for cf
+    ]
+}
+
+rectangle logsearch {
+node "kibana (N x environment)" as kibana {
+    agent kbn [
+        ==kibana
+        The primary interface for viewing logs
+    ]
+    agent kibanaplugin [
+        ==kibana-auth-plugin
```

Comment:
Is this equal to the https://github.com/cloud-gov/kibana-cf-auth-proxy ?

Reply:
no - this is https://github.com/cloud-gov/logsearch-for-cloudfoundry/tree/develop/jobs/kibana-auth-plugin